autospotting.go

```diff
@@ -2,10 +2,11 @@ package main
 
 import (
 	"encoding/json"
-	"flag"
 	"log"
 	"os"
 
+	"github.com/namsral/flag"
```

Comment:
Yes, a go get is unfortunately needed, I'll add it to the Makefile.

I would prefer not to use a vendoring tool for now, especially if it requires all the dependencies to be added to our git repo.